### DISCUSSION TITLE
Want to buy PCB v55 rev 1.45

### DISCUSSION THREAD

**Discussion by THorst92** [nliaudat/esp32_8ch_motor_shield#33] [Buy or Sell]
Hi, 

does anyone have a PCB v55 rev 1.45 (or rev 1.39) for sale?
ESP32 is not needed, the power supply can be ordered and mounted by me.
Shipping would be to Germany.

br 
Thorsten

**Comment by xdubx** [comment]
Hi Thorst92,
i have one laying around with the HLK-5M03 on it. From dez 2022.

**Reply by THorst92** [reply]
Hi, 
I am currently rethinking my setup. Since I only have to operate four actuators, I will possibly design my own control. 
But thanks for the offer!
